kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -990,6 +1052,111 @@ def _run_ovs_cmd(cmd_str):
+            raise Exception(
+                'cannot use NIC for provisioning: '
+                'nic %s has IP address configured and is in use. '
+                'Please remove the IP first: "ip addr flush dev %s"'
+                % (nic_name, nic_name))
+
+        # Check 2: Already an OVS bridge port
+        r, o, e = bash.bash_roe(
+            "ovs-vsctl --if-exists get Interface %s name 2>/dev/null"
+            % nic_name)
+        if r == 0 and o.strip():
+            r2, o2, e2 = bash.bash_roe(
```

Ruff (0.15.9)

[warning] Unpacked variable `e2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)